File: app/src/main/java/com/tencent/test/SquareRect.java
package com.tencent.test;

import android.databinding.ObservableField;

/**
 * Created by Greyzhou on 2018/4/27.
 */

public class SquareRect extends BaseRect{
    public final ObservableField<String> title = new ObservableField<>();
    public void onClick(SquareRect rect){
        title.set("厉害厉害");
    }

}

File: app/src/main/java/com/tencent/test/SquareRect.kt
package com.tencent.test

import android.databinding.ObservableField

/**
 * Created by Greyzhou on 2018/6/29.
 */
class SquareRect : BaseRect(R.layout.layout_square_rect){
    val title = ObservableField<String>()
    val image = ObservableField<String>()
    fun onClick(rect: SquareRect) {
        title.set("厉害厉害")
    }
}